src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -86,6 +88,14 @@ export class FilterController extends Observable {
         return this._addFilterButton;
     }
 
+    get filterCallback() {
+        return this._filterCallback;
+    }
+
+    set filterCallback(_filterCallback) {
```

Comment:
the argument to the function should have an `_` as a prefix, i.e. it should be `set filterCallback(filterCallback) {`